python/pyspark/sql/tests.py

```diff
@@ -1851,6 +1851,71 @@ def test_hivecontext(self):
         self.assertIn("default", out.decode('utf-8'))
         self.assertTrue(os.path.exists(metastore_path))
 
+    def test_hivecontext_create_dataframe(self):
```

Comment:
I think we can just create two `SQLContext` and compare if they share the same `SparkSession` instance.